fairseq/tasks/translation.py

```diff
@@ -492,6 +492,8 @@ def decode(toks, escape_unk=False):
             logger.info("example hypothesis: " + hyps[0])
             logger.info("example reference: " + refs[0])
         if self.cfg.eval_tokenized_bleu:
-            return sacrebleu.corpus_bleu(hyps, [refs], tokenize="none")
+            return sacrebleu.metrics.BLEU().corpus_score(hyps, [refs])
         else:
-            return sacrebleu.corpus_bleu(hyps, [refs])
+            return sacrebleu.metrics.BLEU(trg_lang=self.cfg.target_lang).corpus_score(
```

Comment:
this is probably a good change, but it's also very error prone since it relies on sacrebleu language code. If I use "zho" for my language, I'll get the default tokenization, if I use "zh" I'll get the chinese tokenization.

Reply:
Sorry, I overlooked that. Can a new parameter be added to allow the user to specify the tokenizer that should be used?